src/components/Page/index.ts

```diff
@@ -1,4 +1,3 @@
 import Page, {PageProps} from './Page';
 
 export {Page, PageProps};
-export {HeaderPrimaryAction} from './types';
```

Comment:
This was never used outside of the current component - people should use `PageProps['primaryAction']` instead